src/aap_eda/services/activation/activation_manager.py

```diff
@@ -251,12 +251,7 @@ def _start_activation_instance(self):
             self._error_activation(msg)
             raise exceptions.ActivationStartError(msg) from exc
 
-        # update status
-        self.set_status(ActivationStatus.RUNNING)
-        with transaction.atomic():
-            self.set_latest_instance_status(ActivationStatus.RUNNING)
-            self._set_activation_pod_id(pod_id=container_id)
-        self._reset_failure_count()
+        self._set_activation_pod_id(pod_id=container_id)
```

Comment:
I still thinking that we should not wait for the next cycle of the monitor to move to a running state and it should be  here where we check the readiness as part of the start verb, but I won't block the PR if the rest of team agreed with this approach.

Reply:
As I explained above we don't want 5 seconds to run the monitor. The monitor task runs as soon as the heartbeat is received.

Reply:
> The monitor task runs as soon as the heartbeat is received.

That is not accurate @bzwei   You schedule the monitor task. In your tests it happens fastly because your local env is powerful and mostly idle with 1? 2? 3 concurrent activations? so the workers are idle and everything happens quickly.

In an environment with more workload and/or worse performance, the workers can take much more time to execute both, the monitor or the activation.monitor. 

The point is the fact that you are waiting for the next loop, no matter how fast it happens. It is not only a performance concern but a conceptual one, is the readiness check part of the start flow or the monitor flow?

At the container level, we are not being consistent either. In k8s [we are waiting for it](https://github.com/ansible/eda-server/blob/e05f0359cc859dffb60daa7f371f75b4d3c06270/src/aap_eda/services/activation/engine/kubernetes.py#L496, but we don't do it in podman. 

**As I said, I won't block the PR for this point if the rest of the team agreed with the approach. I don't think it is wrong, Its just I feel it should better to do it as part of the start flow.** ☮️